client/Cargo.toml

```diff
@@ -3,15 +3,16 @@ name = "cosmicverge"
 version = "0.1.0"
 authors = ["Jonathan Johnson <jon@khonsulabs.com>"]
 edition = "2018"
+publish = false
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 [dependencies]
-kludgine = { git = "https://github.com/khonsulabs/kludgine.git", branch = "main", default-features = false, features = ["tokio-rt", "bundled-fonts"] }
+kludgine = { git = "https://github.com/khonsulabs/kludgine.git", default-features = false, features = ["tokio-rt", "bundled-fonts"] }
 tokio = { version = "1.0", features = ["full"] }
-cosmicverge-shared = { path = "../shared", features = ["persyutil"] }
+futures-util = "0.3"
+cosmicverge-shared = { path = "../shared", features = ["persy"] }
 basws-client = "0.1"
 anyhow = "1"
-structopt = "0.3"
+clap = "3.0.0-beta.2"
```

Comment:
I'm not sure I love jumping on a prerelease branch, I see it didn't really take much changing, but I'd prefer to stick to stable releases unless there's a reason not to.

Reply:
There is a good reason, the old version is incredibly old and also incredibly buggy, I have personally not written a lot of CLI tools in Rust yet, and every time I don't upgrade, I find at least a couple of bugs.